packages/paste-website/src/pages/introduction/for-designers/design-guidelines/index.mdx

```diff
@@ -62,129 +62,16 @@ The following are the main libraries we recommend you have enabled for each Figm
+- [Paste Components](https://www.figma.com/file/l2qtpQVMmXzQPVB3rEAA8i/%5BWIP%5D-Paste-Components?node-id=3015%3A46646&t=2Fe524au9O9rlWZU-1) includes the latest components, design tokens (color, text, shadows, spacing, and sizing) in twilio, twilio-dark, default, and evergreen themes. Also [available as a Figma Community file](https://www.figma.com/community/file/1207476064127503112/Twilio-Paste-Components).
+- [Paste Patterns & Templates](https://www.figma.com/file/S4z0Kqjb9AYosnkRQldLte/Paste-Patterns-%26-Templates?type=design&mode=design&t=1bbtpztf0NFxL6kb-0)
+- [Brand Illustrations](https://www.figma.com/file/XeA9WNBcoojrdJbiz5Nj5q/Brand-Illustrations) includes illustrations for empty states, marketing highlights, etc
+- [Asset Documentation](https://www.figma.com/file/GfY11A2VW638eFK48nDtdE/Asset-Documentation) includes helpful utilities for marking up designs, like sticky notes and cursors
 
----
-
-## Using themes
-
-Paste uses 2 theme libraries.
-
-- [Paste Twilio Theme](https://www.figma.com/file/U0oTLE09SGK7svnr818HYT/%5BWIP%5D-Paste-Twilio-Theme?node-id=3558%3A32533&t=PYbe5AovStxUbEjF-1)
-- [Paste Twilio Dark Theme](https://www.figma.com/file/TxxQS4sUuxDiTDUsIWRsFT/Paste-Twilio-Dark-Theme)
-
-When you have a theme library linked, you'll have access to:
-
-- Text styles (font families, font weights, and font sizes)
-- Color styles (fill and stroke for background, border, and text colors)
-- Effect styles (drop shadows, etc.)
-
-<ResponsiveImage src={TokenStyles} alt="Menu in Figma when you can navigate through tokens for the Default theme" />
-
----
-
-## Using components
-
-Components are UI elements that can be reused across your designs. They help you create and manage consistent designs across projects.
-
-You can drag and drop components into your project via the Assets panel on the left, and swap to different variants for a given component via the Design panel on the right.
+#### Twilio community libraries
 
-Check out our [tips and tricks](https://www.figma.com/file/WiJFrYa9YpIcFUmT06gTn0/Working-with-Figma?node-id=155%3A126) for more helpful hints about using the Paste Components library.
+- [Console Library](https://www.figma.com/file/vvdKgsO8OLcn7Q6FmuhWtl/Console-Library) for teams with Console projects, managed by the Console team.
+- [Segment Design Library](https://www.figma.com/file/4tOxLhDIMO2MYvF925RQKc/Segment-Design-Library?type=design&mode=design&t=UNWt1yAShGOocSFf-0) for teams with Segment projects, managed by the Segment team.
+- [File and Cover Sheet Template](https://www.figma.com/file/RfZYRjUH56rZ0qe0veqRJA/File-and-Cover-Sheet-Template)
```

Comment:
suggestion: Do you mind duplicating this callout on this page next to fonts?  or at least something similar.
<img width="748" alt="image" src="https://github.com/twilio-labs/paste/assets/368249/55e60fe5-2ff1-4a92-b22c-cbd32ec59498">